cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -77,15 +77,15 @@ class AdvisorCommand : CliktCommand(name = "advise", help = "Check dependencies
 
     private val globalOptionsForSubcommands by requireObject<GlobalOptions>()
 
-    private val providerFactory by option(
-        "--advisor", "-a",
-        help = "The advisor to use, one of ${allVulnerabilityProvidersByName.keys}"
+    private val providerFactories by option(
+        "--vulnerability-providers", "-v",
```

Comment:
I'm not sure about this renaming. It's also not mentioned as part for the commit message, which makes it sound like a non-breaking changing, although this makes it a breaking change.

From a user-perspective it would anyway be good to not distinguish between vulnerability providers and other advisors here: They just want to be able to enable any kind of advisor by name. So all user-facing output should generally talk about "advisors" IMO.

TL;DR please do not rename this option here. We can revisit this once we add advisors that are not vulnerability providers.

Reply:
Alright, I understand your concerns. What about plural form, i.e. `advisors`?

Reply:
Argh, yes, that's a breaking change that we need 😞 But at least there's a change that people used `-a` before which continues to work...